includes/Modules/Thank_With_Google.php

```diff
@@ -220,6 +221,43 @@ protected function create_data_request( Data_Request $data ) {
 				return function () {
 					return array();
 				};
+			case 'GET:supporter-wall-sidebars':
+				return function() {
+					$sidebars      = array();
+					$all_sidebars  = wp_get_sidebars_widgets();
+					$block_widgets = get_option( 'widget_block' );
+
+					$pattern = sprintf( '/^%s[0-9]+$/i', preg_quote( Supporter_Wall_Widget::WIDGET_ID . '-', '/' ) );
+					$substr  = sprintf( '"idBase":"%s"', Supporter_Wall_Widget::WIDGET_ID );
```

Comment:
This variable is currently unused, but I think it's intended to be used below...

Reply:
Yes, this is needed. Updated.